packages/app/src/app/pages/Dashboard/Components/TeamSubscriptionOptions/TeamSubscriptionOptions.tsx

```diff
@@ -0,0 +1,126 @@
+import { ComboButton, Stack, Text } from '@codesandbox/components';
+import { dashboard as dashboardUrls } from '@codesandbox/common/lib/utils/url-generator';
+import { useGetCheckoutURL } from 'app/hooks/useCreateCheckout';
+import { useWorkspaceAuthorization } from 'app/hooks/useWorkspaceAuthorization';
+import { useWorkspaceSubscription } from 'app/hooks/useWorkspaceSubscription';
+import { useAppState } from 'app/overmind';
+import React from 'react';
+import track from '@codesandbox/common/lib/utils/analytics';
+
+type TeamSubscriptionOptionsProps = {
+  buttonVariant?: React.ComponentProps<typeof ComboButton>['variant'];
+  buttonStyles?: React.ComponentProps<typeof ComboButton>['customStyles'];
+  ctaCopy?: string;
+  trackingLocation: string;
+};
+export const TeamSubscriptionOptions: React.FC<TeamSubscriptionOptionsProps> = ({
+  buttonVariant,
+  buttonStyles,
+  ctaCopy,
+  trackingLocation,
+}) => {
+  const { activeTeam } = useAppState();
+  const { isAdmin } = useWorkspaceAuthorization();
+  const { isFree, isEligibleForTrial } = useWorkspaceSubscription();
+
+  const canSubscribe = isAdmin && isFree && activeTeam !== null;
```

Comment:
If we need to ensure it's also a team and not personal, we can use `isTeamAdmin` instead.

Reply:
@tristandubbeld thank you! I think it's better, updated here: [159eb55](https://github.com/codesandbox/codesandbox-client/pull/7455/commits/159eb5577f930192a714acac7eb2f7b4f39f2aa0).